browser/src/canvas/CanvasSectionContainer.ts

```diff
@@ -2082,13 +2082,16 @@ class CanvasSectionContainer {
 			this.continueAnimating = false;
 		}
 
+		const section: CanvasSectionObject = this.getSectionWithName(this.getAnimatingSectionName());
```

Comment:
I think we don't need to add this. onDraw function is called on animation steps. It looks like moving the last if block to the top of onDraw function would do the same thing (in ScrollSection.ts)? Maybe we can find a way withouth adding this function. Thanks for checking the issue:)

Reply:
This wouldn't do the same thing - this is moving the altering of scroll position outside of the middle of the drawing chain. The problem isn't fixed by altering the scroll position before drawing the scrollbars. The scroll position has external dependencies and needs to be changed before drawing begins, not in the middle of it.

Reply:
The other thing we could do is add an `onBeforeDraw` function, I suppose - the result would be the same, although the impact would be slightly greater (it's negligible either way, but this way, only animated containers are affected, if we add an `onBeforeDraw`, every container will be affected, animating or not).